File: python/processor.py
class processor :
    
    #Atribut
    __nama = ''
    __price = 0

    #konstruktor
    def __init__(self, nama = '', price = 0):
        self.__nama = nama
        self.__price = price

    #prosedur setter data

    def setNama(self, nama):
        self.__nama = nama
    
    def setPrice(self, price):
        self.__price = price

    #prosedur getter data
    def getNama(self):
        return self.__nama
    
    def getPrice(self):
        return self.__price

File: java/Processor.java
public class Processor {
    //atribut
    private String nama;
    private int price;

    //konstruktor
    Processor(){ }

    Processor(String nama, int price){
        this.nama = nama;
        this.price = price;
    }

    //prosedut setter data
    public void setNama(String nama){
        this.nama = nama;
    }

    public void setPrice(int price){
        this.price = price;
    }

    //prosedur getter data
    public String getNama(){
        return this.nama;
    }

    public int getPrice(){
        return this.price;
    }



}
